examples/tags-combinations-go/main.go

```diff
@@ -71,12 +71,40 @@ func main() {
 			return err
 		}
 
+		env, err := appconfig.NewEnvironment(ctx, "testappconfigenv"+testIdent, &appconfig.EnvironmentArgs{
+			ApplicationId: app.ID(),
+			Tags:          tagsMap,
+		}, pulumi.Provider(p))
+		if err != nil {
+			return err
+		}
+
+		getEnv, err := appconfig.GetEnvironment(ctx, "get-testappconfigenv"+testIdent, env.ID(), &appconfig.EnvironmentState{}, pulumi.Provider(p))
+		if err != nil {
+			return err
+		}
+
+		// refresh doesn't work for `forceDelete` & `acl` uncomment when fixed
```

Comment:
This is interesting, is this something worth linking a ticket to? Thank you.

Reply:
Added another comment with link to the issue.